packages/maps/src/lib/mapControlLocationSearch/map-types.ts

```diff
@@ -0,0 +1,14 @@
+import type { Writable } from 'svelte/store';
+
+import type {
+	Map, //
```

Comment:
```suggestion
	Map,
```

Reply:
after removing the empty comment, Prettier might want this import statement on a single line